src/cogent3/core/new_moltype.py

```diff
@@ -424,6 +425,10 @@ def degen_gapped_alphabet(self):
         """monomers + gap + ambiguous characters"""
         return self._degen_gapped
 
+    @property
+    def gaps(self):
+        return frozenset([self.gap, self.missing])
```

Comment:
only add these attributes to the returned value if they are not None. (Some moltypes do not specify them.)